backend/src/controllers/tests/startTest.ts

```diff
@@ -3,15 +3,26 @@ import { Request, Response } from 'express';
 import { TMiddlewareCall } from 'interfaces/commonMiddleware';
 import { isError } from 'utils/typeGuards/isError';
 import { getTestProvider } from 'db/providers/testProvider';
-import { updateCourseStatus } from 'db/providers/clientCourseProvider';
+import { getClientCourseProvider, updateCourseStatus } from 'db/providers/clientCourseProvider';
 import CourseStatus from 'enums/coursesEnums';
+import { IClientCourse } from 'interfaces/Ientities/IclientCourses';
 
 const startTest = async (req: Request, res: Response, next: TMiddlewareCall) => {
   try {
     const { id: clientCourseId } = req.params;
-    const test = await getTestProvider(clientCourseId);
-    await updateCourseStatus(clientCourseId, CourseStatus.testing);
-    res.json(test);
+    const { status, progress }: IClientCourse = await getClientCourseProvider(clientCourseId);
```

Comment:
there is 'getCurrentProgress' provider which return you actual progress of your course
![image](https://user-images.githubusercontent.com/73477329/153561880-bb3b0f16-e30b-447e-9bfb-57ba8b133e6e.png)

Reply:
done